packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -301,8 +420,17 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
       return customItem ? (
         React.cloneElement(customItem, {
           onClick: (e: unknown) => {
```

Comment:
This part could be moved to a variable and reused.

Reply:
Added onClickHandler